src/security/strict-transport-security.conf

```diff
@@ -21,5 +21,5 @@
 # http://blogs.msdn.com/b/ieinternals/archive/2014/08/18/hsts-strict-transport-security-attacks-mitigations-deployment-https.aspx
 
 <IfModule mod_headers.c>
-    Header set Strict-Transport-Security "max-age=16070400; includeSubDomains"
+    Header always set Strict-Transport-Security "max-age=16070400; includeSubDomains"
```

Comment:
FWIW (not directly related to this proposed change) `includeSubDomains` is a terrible default, and easily leads to [why can't I access http://foo.example.com](http://serverfault.com/q/656354/108287) confusion.